### DISCUSSION TITLE
Can't write into entries after clicking "Ok" button of the MessageBox windows

### DISCUSSION THREAD

**Discussion by whyorwhat** [Akascape/CTkMessagebox#44] [Q&A]
`def showConfirmMessage(parent_frame, title, message, icon_image):`
`      CTkMessagebox(master=parent_frame, title=title, message=message, icon=icon_image, fade_in_duration=150)
    `
I've noticed that it works only if I create a frame inside the showConfirmMessage function, but it creates an horrible new window...

**Comment by Akascape** [comment]
Use force_focus method in the entry after closing the messagebox, this issue is related to python environment.
